Assets/MixedRealityToolkit.Services/InputAnimation/IMixedRealityInputRecordingService.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    /// <summary>
+    /// Provides input recording into an internal buffer and exporting to files.
+    /// </summary>
+    public interface IMixedRealityInputRecordingService : IMixedRealityInputDeviceManager
+    {
+        /// <summary>
+        /// Input is being recorded.
+        /// </summary>
+        bool IsRecording { get; }
+
+        /// <summary>
+        /// Limit the size of the recording buffer.
+        /// </summary>
+        /// <remarks>
+        /// If recording is limited any input older than the RecordingBufferTimeLimit will be discarded.
+        /// </remarks>
+        bool UseBufferTimeLimit { get; set; }
```

Comment:
Does this variable and the one below it have any effect after StartRecording has been called? Or is the contract that these only affect the NEXT invocation of StartRecording?

Reply:
These values will affect the recording buffer immediately. If you enable or shorten the limit while recording the excess keyframes are pruned right away. Disabling or extending the limit just allows the buffer to grow over time.